wallet/cardano-wallet.js

```diff
@@ -189,7 +196,25 @@ const CardanoWallet = async (options) => {
     */
     const deviation = 4
 
-    return txFeeFunction(txSizeInBytes + deviation)
+    let fee = txFeeFunction(txSizeInBytes + deviation)
+
+    /*
+    * if they are equal it means there is no change necessary
```

Comment:
this comment is unclear, what is "they"?

Reply:
i thought about rewording this, but hoped this is clear enough, i can add a `(coin+fee)` there for example or word it completely differently